lib/HLSL/DxilGenerationPass.cpp

```diff
@@ -331,9 +331,6 @@ class DxilGenerationPass : public ModulePass {
 
   // Translate precise attribute into HL function call.
   void TranslatePreciseAttribute();
-
-  // Input module is not optimized.
-  bool NotOptimized;
```

Comment:
I see usages of `NotOptimized` in DxcOptimizer.cpp and hctdb.py.
These are not usages of this variable, but they seem to be making the assumption that this class has this member...

Reply:
That's fine. I thought this might be controversial. I'm pretty sure I can add dummy references to this variable to silence the warnings.

Reply:
Thanks :)